tests/functional/lib/helpers.js

```diff
@@ -49,90 +44,68 @@ define(
       );
     }
 
-    // TODO: when https://github.com/mozilla/geckodriver/issues/308 is fixed,
-    // remove this ugliness.
-
-    function login(context) {
-      return openPage(context, url("/login"), "body")
-        .setFindTimeout(config.wc.pageLoadTimeout)
-        .getCurrentUrl()
-        .then(function(url) {
-          // is this the "normal" login flow?
-          if (url.includes("return_to")) {
-            return (
-              context.remote
-                .findByCssSelector("#login_field")
-                .click()
-                .type(config.wc.user)
-                .end()
-                .findByCssSelector("#password")
-                .click()
-                .type(config.wc.pw)
-                .end()
-                .findByCssSelector("input[type=submit]")
-                .click()
-                .end()
-                // *Sometimes* GitHub can bring up an extra verification
-                // page if it detects that our test user is requesting
-                // access too much.
-                .findByCssSelector(".oauth-review-permissions")
-                .then(
-                  function() {
-                    // In this case, there's an extra button to click to convince
-                    // GitHub we're totally not a bot. >_>
-                    return context.remote
-                      .sleep(3000)
-                      .findByCssSelector("button.btn-primary")
-                      .click()
-                      .end();
-                  },
-                  function(err) {
-                    // Otherwise, we swallow the NoSuchElement error.
-                    return new Promise(function(resolve) {
-                      if (/NoSuchElement/.test(String(err))) {
-                        resolve(true);
-                      } else {
-                        throw err;
-                      }
-                    });
-                  }
-                )
-                .end()
-                // ...Now make sure the logged-in avatar is shown so we know we're
-                // back at the home page before we end.
-                .findByCssSelector(".wc-Navbar-avatar")
-                .end()
+    /* 
+    This method makes a call to our API to check that the server is returning fixture data,
+    it will also check if there's anything wrong with the server.
+  */
+    function checkServer() {
+      return new Promise(function(resolve, reject) {
+        var request = http.get(url("/api/issues/100"), function(response) {
+          // If the headers are from non-fixture data,
+          // they will have the "etag" property.
+          if (response.headers.hasOwnProperty("etag")) {
```

Comment:
Another way would be to look for the `_fixture` property on the first object in the array:

https://github.com/webcompat/webcompat.com/blob/03b0da41a8615f160a90529fef51dea64352d0c6/tests/fixtures/api/issues/100/comments.json#L3

(pretty sure I added that to all fixtures manually... )

Reply:
That's cool! I didn't know that. I'll change it :)